projects/orquestra-sdk/src/orquestra/sdk/_client/_base/_driver/_ce_runtime.py

```diff
@@ -652,6 +664,9 @@ def get_workflow_logs(self, wf_run_id: WorkflowRunId) -> WorkflowLogs:
                 f"Failed to decode logs for workflow run with id `{wf_run_id}`. "
                 "Please report this as a bug."
             ) from e
+        except _exceptions.WorkflowRunLogsNotFound:
+            messages = []
+            sys_messages = []
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏what do the CLI logs look like with this change? Is there anything needed to catch this?

Reply:
So I've went with good old consistency.

Basically previously when you had no given type of logs, `orq wf logs <id> --<type> would just produce warning stating logs are empty and nothing else.

Now its the same for all the logs, if you just call `orq wf logs <id>` (notice no type) and there are no logs, you get warning and nothing else.

Reply:
OK great!